public/app/features/trails/ActionTabs/LayoutSwitcher.tsx

```diff
@@ -38,6 +39,7 @@ export class LayoutSwitcher extends SceneObjectBase<LayoutSwitcherState> {
 
   public onLayoutChange = (layout: LayoutType) => {
     reportExploreMetrics('breakdown_layout_changed', { layout });
+    localStorage.setItem(TRAIL_BREAKDOWN_VIEW_KEY, layout);
```

Comment:
Optional: It may be worth considering consolidating all localStorage operations through a common layer, as is done in `TrailStore`.

Then the new `(set/get)BreakdownLayoutType` could be typed & defaulted.

Maybe later as we grow more options like these...

Reply:
Good idea but then I also agree to do this when we really need it.